README.md

```diff
@@ -149,3 +149,30 @@ optional arguments:
   --ignore-invalid-url  Emit a warning but otherwise ignore relative URLs that point to ill-specified locations.
   --local               Write XHTML-based Confluence Storage Format files locally without invoking Confluence API.
 ```
+
+### Using the docker container
+
+You can run the dokcer container via docker run or via dockerfile. Either can accept the environment variables or argument similar to the python options. 
+The final argument `./` is mdpath.
+
+* `docker run --rm --name md2conf hemstreet/md2conf -d instructure.atlassian.net -u levente.hunyadi@instructure.com -a 0123456789abcdef -s DAP ./`
```

Comment:
Assume we will want to make this to match the github repo? so `hunyadi/md2conf`?

Reply:
Good catch, absolutely right.